src/sentry/data_export/endpoints/data_export.py

```diff
@@ -51,6 +51,10 @@ def validate(self, data):
                 detail = f"You can export up to {MAX_FIELDS} fields at a time. Please delete some and try again."
                 raise serializers.ValidationError(detail)
 
+            if "query" not in query_info:
```

Comment:
Can we add an additional condition that `fields` is an non-empty array? Can be another PR though.